src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientGameOverSystem.java

```diff
@@ -86,20 +87,38 @@ public void onGameOver(GameOverEvent event, EntityRef entity) {
     }
 
     private void addPlayerStatisticsInfo(DeathScreen deathScreen) {
-        MigLayout migLayout = deathScreen.find("playerStatistics", MigLayout.class);
-        if (migLayout != null) {
+        MigLayout spadesTeamMigLayout = deathScreen.find("spadesTeamPlayerStatistics", MigLayout.class);
+        MigLayout heartsTeamMigLayout = deathScreen.find("heartsTeamPlayerStatistics", MigLayout.class);
+        if (spadesTeamMigLayout != null && heartsTeamMigLayout != null) {
             Iterable<EntityRef> characters = entityManager.getEntitiesWith(PlayerCharacterComponent.class, LASTeamComponent.class);
             for (EntityRef character : characters) {
                 EntityRef client = character.getOwner();
                 ClientComponent clientComponent = client.getComponent(ClientComponent.class);
-                migLayout.addWidget(new UILabel(PlayerUtil.getColoredPlayerName(clientComponent.clientInfo)), new MigLayout.CCHint());
+                String playerTeam = localPlayer.getCharacterEntity().getComponent(LASTeamComponent.class).team;
                 PlayerStatisticsComponent playerStatisticsComponent = character.getComponent(PlayerStatisticsComponent.class);
-                migLayout.addWidget(new UILabel(String.valueOf(playerStatisticsComponent.kills)), new MigLayout.CCHint());
-                migLayout.addWidget(new UILabel(String.valueOf(playerStatisticsComponent.deaths)), new MigLayout.CCHint("wrap"));
+                MigLayout migLayout = (playerTeam.equals("black") ? spadesTeamMigLayout : heartsTeamMigLayout);
+                addInfoToTeamMigLayout(migLayout,clientComponent,playerStatisticsComponent);
             }
         }
+        addTeamScore(deathScreen);
     }
-    
+
+    private void addInfoToTeamMigLayout(MigLayout migLayout, ClientComponent clientComponent,PlayerStatisticsComponent playerStatisticsComponent){
+        migLayout.addWidget(new UILabel(PlayerUtil.getColoredPlayerName(clientComponent.clientInfo)), new MigLayout.CCHint());
+        migLayout.addWidget(new UILabel(String.valueOf(playerStatisticsComponent.kills)), new MigLayout.CCHint());
+        migLayout.addWidget(new UILabel(String.valueOf(playerStatisticsComponent.deaths)), new MigLayout.CCHint("wrap"));
+    }
+
+    private void addTeamScore(DeathScreen deathScreen){
+        UILabel spadesTeamScore = deathScreen.find("spadesTeamScore", UILabel.class);
+        UILabel heartsTeamScore = deathScreen.find("heartsTeamScore", UILabel.class);
+        ClientScoreSystem clientScoreSystem = new ClientScoreSystem();
+        String blackScore = clientScoreSystem.getBlackScore();
+        String redScore = clientScoreSystem.getRedScore();
+        spadesTeamScore.setText(blackScore);
+        heartsTeamScore.setText(redScore);
+    }
```

Comment:
Here I get the score number of `0` for both teams. I believe this is probably because the score elements get reset somewhere in between when the game ends and when the scores get showcased.
![image](https://user-images.githubusercontent.com/48293545/89718871-65ef6e00-d9cb-11ea-960f-df4deb452a4c.png)